visualstudioextension.ruleset

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <RuleSet Name="Visual Studio Extension rules" ToolsVersion="17.0">
-  <Include Path="general.ruleset" Action="Default" />
+  <Include Path="General.ruleset" Action="Default" />
```

Comment:
Is the capital letter on purpose here? Everywhere else you've changed the name to lower case.